File: Program-Design/Python/Python-100/n21.py
a=int(input())
b=int(input())
c=int(input())
if a<b and b<c :
    if pow(a,2)+pow(b,2)==pow(c,2):
        print("Right triangle")
    elif pow(a,2)+pow(b,2)<pow(c,2):
        print("Obtuse triangle")
    elif pow(a,2+pow(b,2))>pow(c,2):
        print("Acute triangle")

File: Program-Design/Java/Java-100/n21.java
import java.util.Scanner;

public class n21 {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int a =scn.nextInt();
        int b =scn.nextInt();
        int c =scn.nextInt();
        if(a*a+b*b==c*c){
            System.out.println("Right triangle");
        }
        else if(a*a+b*b<c*c){
            System.out.println("Obtuse triangle");
        }
        else if(a*a+b*b>c*c){
            System.out.println("Acute triangle");
        }
    }
}
